src/GregClient/PackageManagerClient.cs

```diff
@@ -0,0 +1,144 @@
+﻿using RestSharp;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Greg.Requests;
+using Greg.Responses;
+using System.Net;
+using System.IO;
+using Greg.Utility;
+using Newtonsoft.Json;
+
+namespace Greg
+{
+    /// <summary>
+    /// Package Manager REST Client for ACG API
+    /// </summary>
+    public class PackageManagerClient : IGregClient
+    {
+        private const string AFC_CODE = "DY1ON1";
+        private readonly RestClient _client;
+        private readonly RestClient _fileClient;
```

Comment:
What is fileClient, what's the difference between _client and _fileClient? Please document it.